app/classifier/drawing-tools/ellipse.cjsx

```diff
@@ -53,6 +56,13 @@ module.exports = React.createClass
     x: @props.mark.rx * Math.cos theta
     y: -1 * @props.mark.ry * Math.sin theta
 
+  dragEnd: (target) ->
+    @context.geordi?.logEvent type: 'drag-ellipse'
+    deleteIfOutOfBounds target
+
+  logResize: ->
+    @context.geordi?.logEvent type: 'resize-ellipse'
+
```

Comment:
These handlers are being called on drag end, rather than during the tool resize. I don't think that will generate a huge volume of data.